proto/manager/firmware_update.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+import "error.proto";
+import "common.proto";
+
+package manager;
+
+/**
+ * Example of firmware update confirmation
+ *
+ *        Host                       Device
+ * FirmwareUpdateInitiateRequest  => 
+ *                                <= FirmwareUpdateConfirmed
+ */
+
+/**
+ * Example of firmware update rejection
+ *
+ *        Host                       Device
+ * FirmwareUpdateInitiateRequest  => 
+ *                                <= error.CommonError
+ */
+ 
+message FirmwareUpdateInitiateRequest {
+  common.Version version = 1;
+}
+
+message FirmwareUpdateRequest {
```

Comment:
keep this just above `FirmwareUpdateResponse`

just a pattern for ordering the structs in the file.

Reply:
updated